mevislab.github.io/content/examples/image_processing.md

```diff
@@ -13,6 +13,6 @@ menu:
 ## Image Processing Examples:
 [Arithmetic Operations](/examples/image_processing/example1/)  
 [Masking Images](/examples/image_processing/example2/)  
-[Region Growing (Segmentation)](/examples/image_processing/example3/) 
-[Subtract 3D objects](/examples/image_processing/example4/) 
+[Region Growing (Segmentation)](/examples/image_processing/example3/)  
+[Subtract 3D objects](/examples/image_processing/example4/)  
```

Comment:
Is there only a space added and line ends?

Reply:
yes, without spaces, linebreaks in HTML are missing, strange behaviour :(

Reply:
Ah, true - this is standard markdown syntax, 2 spaces denote a line break

Reply:
cant we leave it like this? Its standard md and works :)